### PR TITLE
Platform independent compilation

### PR DESCRIPTION
**Pull request by heniu20**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -598,13 +598,42 @@
...
+                <condition property="urlSlashesOsModifier" value="//">
+                  <and>
+                    <not>
+                      <os family="windows"></os>
+                    </not>
+                  </and>
+                </condition>
+                <condition property="urlSlashesOsModifier" value="///">
+                  <and>
+                    <os family="windows"></os>
```
Comment (velo) [inline pom.xml:622]: ow windows.... gotta love that.

Funny enough, I'm on windows 11, but all work is done under WSL, so never noticed the problem

File: pom.xml [hunk 2]
```diff
@@ -598,13 +598,42 @@
           <stagingProgressTimeoutMinutes>15</stagingProgressTimeoutMinutes>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
```
Comment (velo) [inline pom.xml:602]: can we use maven profiles instead?

`ant` seems so 1994

File: querydsl-codegen/src/test/java/com/querydsl/codegen/BeanSerializerTest.java [hunk 3]
```diff
@@ -134,7 +134,8 @@ public void toString_() throws IOException {
     BeanSerializer serializer = new BeanSerializer();
     serializer.setAddToString(true);
     serializer.serialize(type, SimpleSerializerConfig.DEFAULT, new JavaWriter(writer));
-    assertThat(String.valueOf(writer)).contains("    @Override\n" + "    public String toString()");
```
Comment (velo) [inline querydsl-codegen/src/test/java/com/querydsl/codegen/BeanSerializerTest.java:137]: would a multine string fix this?

```
assertThat(String.valueOf(writer)).contains("""
 @Override
 public String toString()
""");
```

Comment (heniu20) [reply querydsl-codegen/src/test/java/com/querydsl/codegen/BeanSerializerTest.java:137]: Nope, text blocks always use \n

File: pom.xml (hunk 2 above)
Comment (heniu20) [reply pom.xml]: done

**Review by velo — APPROVED** [review]

**Comment by velo** [comment]
I made a few changes to eliminate the need of a maven profile and asserting line endings...
Also enable windows buld

Lemme know what you think.

**Pull request merged**